CONTRIBUTORS

```diff
@@ -0,0 +1,122 @@
+8cbx <8cbx@users.noreply.github.com>
```

Comment:
Yes. I already try my best to filter out some invalid email address. I need user identify themselves.